string_manipulation.rb

```diff
@@ -50,9 +50,26 @@ def reverse_sentence(my_sentence)
 # A method to check if the input string is a palindrome.
 # Return true if the string is a palindrome. Return false otherwise.
 def palindrome_check(my_phrase)
-  puts "NOT IMPLEMENTED"
+  i = 0
+  j = my_phrase.length - 1
+
+  until i >= j
+    if my_phrase[i] == " "

```

Comment:
Looks great! 
As an added optimization, consider making the conditional statements into loops for skipping white spaces in my_phrase - just to make your method even more flexible.